src/graphapi/engineheur.cpp

```diff
@@ -197,7 +197,7 @@ void BackendEngineHeurDescriptor::getAttribute(miopenBackendAttributeName_t attr
         {
             *elementCount = mResults.size();
             std::transform(mResults.begin(),
-                           mResults.begin() + std::min(*elementCount, requestedElementCount),
```

Comment:
@apwojcik it should be fixed in Clang, right? Is the compiler team aware of this issue?